pretrain/openwebtext.py

```diff
@@ -158,30 +158,30 @@ def train(fabric: L.Fabric, state: dict, train_dataloader: DataLoader, val_datal
             optimizer.zero_grad()
             state["step_count"] += 1
 
-        total_lengths += input_ids.size(1)
```

Comment:
The pretraining and XLA scripts do not need to track this per batch because they are all of the same size